### DISCUSSION TITLE
Innova MQTT

### DISCUSSION THREAD

**Discussion by FelixB22** [danielrivard/homeassistant-innova#143] [General]
Hello Daniel
Thanks for your excellent work for the Innova 2.0 / Home Assistant.
I think, the Innova device will send his data to a MQTT Broker ?
Is this correct and is this Broker known and available for use ?
regards,
Felix

**Comment by danielrivard** [comment]
Hi, by default, no, the Innova integration doesn't send to MQTT broker, it will just send events to Home Assistant through the integration. I am not very familiar with MQTT Brokers so I don't think I can help much.
